cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -840,7 +840,8 @@ public void searchMetadata(HttpRequest request, HttpResponder responder,
                              // 2147483647 is Integer.MAX_VALUE
                              @QueryParam("limit") @DefaultValue("2147483647") int limit,
                              @QueryParam("numCursors") @DefaultValue("0") int numCursors,
-                             @QueryParam("cursor") @DefaultValue("") String cursor) throws Exception {
+                             @QueryParam("cursor") @DefaultValue("") String cursor,
+                             @QueryParam("showHidden") boolean showHidden) throws Exception {
```

Comment:
Since its a boolean it defaults to false. Please see conversation here: https://github.com/caskdata/cdap/pull/7439#discussion_r92915860
It's not always required. If not specified it will be false.